doc/sphinx/source/frameworks/index.rst

```diff
@@ -1,14 +1,15 @@
-.. optimize/index: 
+.. framework/index: 
 
 #############################
 Integrate Generic Frameworks   
 #############################
 
-This section, written for framework architects or engineers who want 
-to optimize a generic, brand new or less widely-supported framework, we
-provide some of our learnings from the work we've done in developing 
-"framework direct optimizations (DO)" and custom bridge code, such as 
-that for our `ngraph tensorflow bridge`_ code.
+In this section, written for framework architects or engineers who want 
+to optimize a brand new, generic, or less widely-supported framework, we provide 
```

Comment:
frameworks

Reply:
> To optimize a(n) `a`, `b`, or `c` [class/category/kind of] framework ... 

is technically correct, but I'll nuke the article.